src/blockmap.jl

```diff
@@ -24,8 +24,8 @@ BlockMap{T}(maps::As, rows::Rs) where {T, As<:LinearMapTuple, Rs} =
 
 MulStyle(A::BlockMap) = MulStyle(A.maps...)
 
-function _getranges(maps, dim, inds::NTuple{N,Int}=ntuple(identity, Val(length(maps)))) where {N}
-    sizes = ntuple(i -> (@inbounds size(maps[inds[i]], dim)), Val(N))
+function _getranges(maps, dim, inds=ntuple(identity, Val(length(maps))))
+    sizes = ntuple(i -> (@inbounds size(maps[inds[i]], dim))::Int, Val(length(inds)))
```

Comment:
why not `sizes = map(i->size(maps[i], dim), inds)`?

Reply:
I'm not sure I have tried this one, but I'm testing many different details, which behave differently in view of type inference in the wide range of Julia versions.